.gitignore

```diff
@@ -104,4 +104,7 @@ dist
 .tern-port
 
 # JS files
-*.js
\ No newline at end of file
+*.js
+
+# TODO files
+routes.ts
```

Comment:
If you have a file that's still TODO, the best move is just not to commit it. It's a lot of work adding and removing files from gitignore over and over, and if you ever forget, it's a huge faff.

As a general rule, the gitignore is for things which you are sure should be permanently ignored